hi-ml-cpath/src/health_cpath/preprocessing/tiff_conversion.py

```diff
@@ -214,7 +214,10 @@ def get_px_per_cm_resolution_at_level(self, wsi_obj: OpenSlide, level: int) -> T
         :return: A tuple of floats (x_resolution, y_resolution)
         """
         um_per_cm = 10000
-        um_per_px = self.wsi_reader.get_mpp(wsi_obj, level=level)
+        try:
+            um_per_px = self.wsi_reader.get_mpp(wsi_obj, level=level)
+        except KeyError:
+            um_per_px = (wsi_obj.properties["openslide.mpp-x"], wsi_obj.properties["openslide.mpp-y"])
```

Comment:
cover this in test (codecov warning)

Reply:
we don't have mock svs files to test this